operator/duties/committee.go

```diff
@@ -12,23 +12,29 @@ import (
 	"github.com/ssvlabs/ssv/operator/duties/dutystore"
 )
 
-type committeeDutiesMap map[spectypes.CommitteeID]*spectypes.CommitteeDuty
+type committeeDutiesMap map[spectypes.CommitteeID]*committeeDuty
 
 type CommitteeHandler struct {
 	baseHandler
 
 	attDuties  *dutystore.Duties[eth2apiv1.AttesterDuty]
 	syncDuties *dutystore.SyncCommitteeDuties
 
-	validatorCommitteeIDs map[phase0.ValidatorIndex]spectypes.CommitteeID
+	validatorCommitteeDutyMap map[phase0.ValidatorIndex]*committeeDuty
+}
+
+type committeeDuty struct {
```

Comment:
Do I understand correctly that 
- we never use `duty` in `validatorCommitteeDutyMap`, although the type is called `committeeDuty`;
- `id` is redundant in `committeeDutiesMap`?

Should we perhaps use different types for them?

Reply:
the `duty` is indeed used in the `ExecuteCommitteeDuties` function, where it's executed via `committee.duty`.

As for the `id` field in `committeeDutiesMap`,  it simplifies access and lookup by avoiding recalculating or passing it around during the building of duties, improving performance.